torch/nested/_internal/ops.py

```diff
@@ -730,6 +755,20 @@ def native_layer_norm_default(func, *args, **kwargs):
     return (NestedTensor(output, **extract_kwargs(inp)), mean, std)
 
 
+@register_jagged_func(
+    torch.ops.aten.native_layer_norm_backward.default,
+    "grad_out: jt, input: jt, normalized_shape: any, mean: any, rstd: any, weight: any?, bias: any?, output_mask: any",
+)
+def native_layer_norm_default(func, *args, **kwargs):
+    _, new_kwargs = normalize_function(
+        func, args=args, kwargs=kwargs, normalize_to_only_use_kwargs=True
+    )
+    grad_out = new_kwargs.pop("grad_out")
+    inp = new_kwargs.pop("input")
+    d_input, d_gamma, d_beta = func(grad_out._values, inp._values, **new_kwargs)
+    return (NestedTensor(d_input, **extract_kwargs(inp)), d_gamma, d_beta)
```

Comment:
We should check if `d_input` is None here in the case where input does not require grad.